odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/mapper/DatasetVersionApiMapper.java

```diff
@@ -0,0 +1,64 @@
+package org.opendatadiscovery.oddplatform.mapper;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.opendatadiscovery.oddplatform.api.contract.model.DataSetField;
+import org.opendatadiscovery.oddplatform.api.contract.model.DataSetStructure;
+import org.opendatadiscovery.oddplatform.api.contract.model.DataSetVersion;
+import org.opendatadiscovery.oddplatform.dto.DatasetFieldDto;
+import org.opendatadiscovery.oddplatform.dto.DatasetStructureDto;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.DatasetFieldPojo;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.DatasetVersionPojo;
+
+@Mapper(config = MapperConfig.class, uses = {
+    DatasetFieldApiMapper.class
+})
+public interface DatasetVersionApiMapper {
```

Comment:
You created new mapper, but I don't see that you removed an old one
And let's call it just `DatasetVersionMapper`